data/about.tsx

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+type Content = {
+  [key: string]: React.ReactNode;
+};
+
+export const aboutAmcovad = (
+  <>
+    <p className="paragraphs">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Diam volutpat,
+      rutrum bibendum tincidunt nec velit dictumst. Pellentesque Luspendisse
+      ultricies at a nunc, morbi. Amet, varius pulvinar quam fermentum, nulla
+      nulla cras aliquam leo. Sit facilisis in posuere gravida. Vitae at risus
+      urna tempor eu vulputate id scelerisque lorem. Auctor sagittis, egestas
+      viverra tincidunt elit. Volutpat bibendum facilisis sed quam et. At
+      viverra nibh vestibulum, vitae faucibus elementum pharetra viverra ut. Et
+      arcu mi volutpat ornare donec lorem. Velit imperdiet porta purus amet
+      aliquet sodales egestas condimentum vivamus. Eros lacus cras penatibus nec
+      donec elementum tristique.
+    </p>
+
+    <p>
+      Nec id proin vitae praesent et egestas libero, mauris sem. Feugiat
+      vulputate gravida dui viverra diam justo, iaculis. Ut amet, volutpat, ut
+      suspendisse scelerisque sit proin duis. Hendrerit id vitae sapien, sit
+      pellentesque tempor ultrices. Luctus mollis vitae urna, a. In iaculis
+      laoreet sit proin nisi, enim cursus est arcu. Ullamcorper suscipit
+      pretium, ac malesuada amet velit. Quam gravida aliquet dignissim cursus
+      tincidunt sagittis, aliquet magna posuere. Egestas faucibus elit at
+      mauris, metus, mauris, fermentum.{" "}
+    </p>
+
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Potenti nullam ac
+      tortor vitae. Donec pretium vulputate sapien nec sagittis aliquam
+      malesuada bibendum. Ut eu sem integer vitae justo eget magna fermentum
+      iaculis. Eget mi proin sed libero. Aliquet enim tortor at auctor urna nunc
+      id cursus. A scelerisque purus semper eget duis at tellus at urna.
+      Facilisis gravida neque convallis a cras semper. Dolor sit amet
+      consectetur adipiscing elit duis tristique sollicitudin nibh. Faucibus
+      scelerisque eleifend donec pretium vulputate sapien. Volutpat est velit
+      egestas dui id ornare arcu odio.
+    </p>
+
+    <p>
+      Tellus molestie nunc non blandit massa enim. Gravida dictum fusce ut
+      placerat orci nulla. Aliquam vestibulum morbi blandit cursus. Nisl
+      condimentum id venenatis a condimentum. Sagittis id consectetur purus ut
+      faucibus pulvinar. Enim nunc faucibus a pellentesque sit amet porttitor
+      eget. Mus mauris vitae ultricies leo integer malesuada. Nunc sed id semper
+      risus in hendrerit gravida rutrum.
+    </p>
+  </>
+);
+export const ourMission = (
+  <>
+    <p className="paragraphs">
```

Comment:
This should be [dropcap](https://www.google.com/search?q=drop+cap&rlz=1C5CHFA_enNG925NG925&sxsrf=ALiCzsYttWT1cywP8FNIa0aZBK6fQKz6KQ%3A1664250789202&ei=pXMyY8r6C4OolwTmnLaoDg&ved=0ahUKEwiK4e2oibT6AhUD1IUKHWaODeUQ4dUDCA4&uact=5&oq=drop+cap&gs_lcp=Cgdnd3Mtd2l6EAMyBQgAEJECMgQIABBDMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQkQIyBQgAEJECMgUIABCABDIFCAAQgAQ6BAgjECc6BQguEJECOggIABCABBCxAzoFCAAQsQM6BQguELEDOg0ILhCxAxDHARDRAxBDOgoIABCxAxCDARBDOgsIABCABBCxAxCDAToICC4QgAQQsQM6BwgAELEDEEM6BggAEAoQQ0oECEEYAEoECEYYAFAAWNkHYMQLaABwAXgAgAGDAogBsw2SAQUwLjEuN5gBAKABAcABAQ&sclient=gws-wiz) not paragraphs. 

```suggestion
    <p className="dropcap">
```